app/src/organisms/ProtocolSetupParameters/index.tsx

```diff
@@ -85,6 +94,15 @@ export function ProtocolSetupParameters({
     }
   }
 
+  const runTimeParameterValues = getRunTimeParameterValuesForRun(
+    runTimeParametersOverrides
+  )
+  const { createProtocolAnalysis } = useCreateProtocolAnalysisMutation(
+    {},
+    protocolId,
+    host
+  )
```

Comment:
If we change like below

```ts
export function useCreateProtocolAnalysisMutation(
  protocolId: string | null,
  hostOverride?: HostConfig | null,
  options: UseCreateProtocolAnalysisMutationOptions | undefined = {}
): UseCreateProtocolMutationResult {
```

We don't need to pass `{}`.

```suggestion
  const { createProtocolAnalysis } = useCreateProtocolAnalysisMutation(
    protocolId,
    host
  )
```

Reply:
smart. fixing that.